### PR TITLE
set up webpack and npm

### PR DESCRIPTION
**Pull request by JYC-99**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+var webpack = require('webpack');
+
+module.exports = {
+  devtool: 'source-map',
+  entry: [
+    'react-hot-loader/patch',
+    './src/index.js',
+  ],
+  output: {
```
Comment (wewei) [inline webpack.config.js:11]: Output as UMD/AMD?

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+
+var webpack = require('webpack');
+
```
Comment (wewei) [inline webpack.config.js:4]: How do you build demo?

Comment (wewei) [reply webpack.config.js:4]: DEMO should reference the output in `dist`.

**Review by wewei — APPROVED** [review]

File: webpack.config.js (hunk 1 above)
Comment (JYC-99) [reply webpack.config.js:11]: Fixed

File: webpack.config.js (hunk 2 above)
Comment (JYC-99) [reply webpack.config.js:4]: Not sure if hot-reload still work... Should leave it as it is now and investigate later.

**Pull request merged**